extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -159,6 +166,15 @@ export class DraggableDrawer extends AMP.BaseElement {
       this.containerEl_.insertBefore(spacerEl, this.contentEl_);
       this.contentEl_.appendChild(headerShadowRootEl);
       this.element.classList.add('amp-story-page-attachment-ui-v2');
+
+      // Remove background and backgroundColor from parent and put on content element.
+      const bgColor = computedStyle(this.win, this.element)['backgroundColor'];
+      const bg = computedStyle(this.win, this.element)['background'];
+      setStyles(this.contentEl_, {
+        'backgroundColor': bgColor,
```

Comment:
Ok for proof of concept, but I wonder if it's better for us to maintain an allowlist of properties, all of which will be copied?